server.js

```diff
@@ -1,8 +1,45 @@
 const express = require('express');
 const path = require('path');
-
+const Raven = require('raven');
 const app = express();
 
+// Must configure Raven before doing anything else with it
+Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();
+
+// The request handler must be the first middleware on the app
+app.use(Raven.requestHandler() {
+  parseUser: function (req) {
+    // custom user parsing logic
+    return {
+      username: req.users.username,
+      id: req.user.getId()
+     };
+   }
+});
+
+app.get('/', function mainHandler(req, res) {
+    throw new Error('Broke!');
```

Comment:
What's the output when we receive an error? Is there anything displayed on the server or client side?